### ISSUE TITLE
Extract method sometimes doesn't add a return statement

### ISSUE THREAD

**Issue by haferburg** [microsoft/pylance-release#2906]
In the line `a = "hello"` select the "hello", extract method.

```
def new_func():
    return "hello"

a = new_func()
```

Now select the `new_func()` in the last line.

```
def new_func():
    return "hello"

def new_func1(new_func):
    new_func()

a = new_func1(new_func)
```

I would have expected a different output:

```
def new_func():
    return "hello"

def new_func1():
    return new_func()

a = new_func1()
```

-------------

Similarly, when I select the function call here

_res =_ **p4.run("unsubmit", f"//...@{cl}")**

and extract method, it creates this:

```
res = new_func2(p4, cl)

def new_func2(p4, cl):
    p4.run("unsubmit", f"//...@{cl}")
```

I would have expected
```
res = new_func2(p4, cl)

def new_func2(p4, cl):
    return p4.run("unsubmit", f"//...@{cl}")
```

-----------------

> Read the guidelines for filing an issue first.
> https://github.com/microsoft/pylance-release/blob/master/TROUBLESHOOTING.md#filing-an-issue
> Copy the entire log starting with "Pylance language server XXX (pyright xxx) starting"

Sorry, I'm not going to do that. It's way too long and I don't want to review all of it.

    [Info  - 13:21:42] (4184) Pylance language server 2022.6.10 (pyright bbc42826) starting

Please add a command that collects all the data you need for bug reports, or log it in the first couple lines.

**Comment by heejaechang** [comment]
looks like extract method doesn't recognize the selection as expression but as statement and don't put "return"

**Comment by heejaechang** [comment]
This issue has been fixed in version 2022.7.41, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022741-20-July-2022)
